ai-ml/trainium-inferentia/addons.tf

```diff
@@ -267,7 +267,7 @@ module "eks_data_addons" {
   #---------------------------------------
   # Kuberay Operator
   #---------------------------------------
-  enable_kuberay_operator = true
+  enable_kuberay_operator = var.enable_kuberay_operator
```

Comment:
Are you disabling Kuberay operator as a part of the workshop?

Reply:
yes @vara-bonthu